package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "cordova-sqlite-storage",
-  "version": "3.2.0-OS3",
-  "description": "Native interface to SQLite for PhoneGap/Cordova - cordova-sqlite-storage plugin version",
+  "version": "6.0.0-OS1",
```

Comment:
According to the new specification this should not use the `-OSX` suffix but instead use `+<semver>` suffix. Can be fixed here or in a posterior raise commit (which will be tagged).